import-service/handler.js

```diff
@@ -85,12 +91,19 @@ async function processCsvFile(bucket, key) {
     const records = [];
 
     const readStream = s3.getObject({ Bucket: bucket, Key: key }).createReadStream();
-
     readStream
       .pipe(csvParser({ separator: ';' }))
       .on('data', (data) => {
         data.price = Number(data.price);
+        data.id = data.id;
         records.push(data);
+        sqs.sendMessage({
```

Comment:
Yep, it works. But good catch, it might be an issue with a big amount of data. Fixed.

Reply:
Yeh, got it why it works - here we only send to sqs queue, and a product is created within catalogBatchProcess. But still added await as it can affect logging in the importFileParser.

Reply:
Okay